src/Middleware/RateLimiting/test/RateLimitingMetricsTests.cs

```diff
@@ -228,7 +232,11 @@ public async Task Metrics_Queued()
             m => AssertCounter(m, 1, "concurrencyPolicy"),
             m => AssertCounter(m, -1, "concurrencyPolicy"));
         Assert.Collection(queuedRequestDurationCollector.GetMeasurementSnapshot(),
-            m => AssertDuration(m, "concurrencyPolicy"));
```

Comment:
Design nit: The design doesn't specify the list of policy strings that I saw, but in general OTel seems to favor snake_case naming rather than camelCase. I don't care, but just mentioning it in case others feel it is important?

Reply:
if I understand correctly, policy name is an arbitrary string set by a user in the code, so it's perfectly fine to ignore otel conventions here and match user input

Reply:
Correct